src/wallet/coincontrol.h

```diff
@@ -64,6 +65,8 @@ class CCoinControl
     int m_min_depth = DEFAULT_MIN_DEPTH;
     //! Maximum chain depth value for coin availability
     int m_max_depth = DEFAULT_MAX_DEPTH;
+    //! ELEMENT: Override the chain's default fee asset if set
```

Comment:
ELEMENT? Or ANY_ASSET_FEES ?

Reply:
Rewrote the comment in: https://github.com/SequentiaSEQ/SEQ-Core-Elements/pull/10/commits/15f8801f8737c850873f6e237af459b2a90b1637

Also prevented the variable from being set when any asset fees isn't enabled.